### ISSUE TITLE
Allow RMD Admin to Upload to SS 

### ISSUE THREAD

**Issue by DanCoughlin** [psu-stewardship/researcher-metadata#194]
RMD Admins receive emails from faculty with attachments that include the post-print for compliance with the OA Policy. Admins should be able to log into RMD and upload the paper to ScholarSphere (similar to the integration that exists for researchers).

**Comment by awead** [comment]
@anaelizabethenriquez Yes on both points. The sticking point in my mind is that the publication data has already been imported, so we're just taking it the last step.

**Comment by anaelizabethenriquez** [comment]
OK, thanks for this additional detail, @ajkiessl.
